app/controllers/people_filters_controller.rb

```diff
@@ -82,7 +77,7 @@ def people_list_path(options = {})
   end
 
   def load_possible_tags
-    @possible_tags ||= Person.tags
+    @possible_tags ||= PersonTags::Translator.new.possible_tags
```

Comment:
can't see any difference in numbers of loaded entries here.
since we're using tags only for people, it should not load more than before.
Am I missing something ?